### PR TITLE
Add new teleport locations to world map

### PR DESCRIPTION
**Pull request by forsco**
Implements the foundation files for miscellaneous teleport locations.

Xeric's Talisman teleport locations added to the world map plugin.

Fixes: #3324

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MiscTeleportLocationData.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  * Copyright (c) 2018, forsco <https://github.com/forsco>
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MiscTeleportLocationData.java]: There should only be one * on these lines

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MiscTeleportPoint.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ *  * Copyright (c) 2018, forsco <https://github.com/forsco>
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MiscTeleportPoint.java]: Header *'s

**Comment by MESLewis** [comment]
You can use git commit --fixup to keep your history clean of little fixes

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java [hunk 3]
```diff
@@ -61,6 +61,7 @@
 	static final String CONFIG_KEY_LUNAR_TELEPORT_ICON = "lunarSpellbookIcon";
 	static final String CONFIG_KEY_ARCEUUS_TELEPORT_ICON = "arceuusSpellbookIcon";
 	static final String CONFIG_KEY_JEWELLERY_TELEPORT_ICON = "jewelleryIcon";
+	static final String CONFIG_KEY_MISC_TELEPORT_ICON = "miscellanwousTeleportIcon";
```
Comment (forsco) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java]: "miscellanwousTeleportIcon" will be corrected in a future push.

**Comment by forsco** [comment]
Thanks @MESLewis, Ill do a commit squash once I'm finished adding in new locations.

**Comment by forsco** [comment]
I'm not going to have time through out the week days to work on this. It can be merged if accepted so others can possibly extend on it.

**Comment by Adam-** [comment]
I refactored the plugin to be easier to add new teleports to in #3391 - and included this in to that.

**Pull request closed without merging**